controllers/url_controller.js

```diff
@@ -0,0 +1,62 @@
+import Url from "../models/Url.js";
+
+const shorten_url = async (req, res, next) => {
+    try{
+        let { original_url } = req.body;
+
+        if (!original_url) {
+            return res.status(400).send({ error: 'A URL is required.' });
```

Comment:
We can make the error message a bit more specific for API users. An example would be 'An original_url body parameter is required'